markdown/extensions/toc.py

```diff
@@ -295,6 +297,9 @@ def __init__(self, **kwargs):
                         "Function to generate anchors based on header text - "
                         "Defaults to the headerid ext's slugify function."],
             'separator': ['-', 'Word separator. Defaults to "-".'],
+            "toc_height": [1,
+                          "Define from to which section level n (<hn>..<h6>) to "
+                          "include in the TOC"],
```

Comment:
There must be a typo here. "from to which" can' the correct. I'm not even sure what it is supposed to say.

Reply:
Also, you are getting a [flake8 error][1] here. Note that the [Contributing Guidelines][2] define how to format multi-line lists. I realize the other items in the file don't match that style, but they predate the guidelines, and as the guiltiness state:

> Legacy code which does not follow the guidelines should only be updated if and when other changes (bug fix, feature addition, etc.) are being made to that section of code. 

[1]: https://travis-ci.org/Python-Markdown/markdown/jobs/490761623
[2]: https://github.com/Python-Markdown/markdown/blob/master/docs/contributing.md#code-style-guide